deploy/helm/nfd-operator/crds/nfd-api-crds.yaml

```
@@ -0,0 +1,687 @@
+---
+---
```

nit: extra separator 

```suggestion
---
```